### DISCUSSION TITLE
Sorc no tele

### DISCUSSION THREAD

**Discussion by FakeForFearofFlame** [isid0re/SoloLeveling#62] [Q&A]
Sorc is level 28 now has respec'd once and stated tele as a skill but wont use it.

Help??

**Comment by FakeForFearofFlame** [comment]
> Sorc is level 28 now has respec'd once and stated tele as a skill but wont use it.
> 
> Help??

Also stuck in norm just doing pit & tomb runs :/

**Reply by isid0re** [reply]
Ok most likely, doesnt have enough gold to support buying mana potions. At lower levels teleport consumes lots of mana. In normal that is 10k, 50k nightmare, 100k in hell.  

So the bot disables teleport till it has enough gold to maintain buying pots  or mana level to support consistant teleport. If not it will just stand there and get hit by mobs and not move or attack.

It will run tombs and pits for gold, sounds like you had a bit of bad rng.
